iOS-Settings/Sources/App/DetailViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DetailViewController.swift
+//  iOS-Settings
+//
+//  Created by Kuanysh al-Khattab Auyelgazy on 03.03.2023.
+//
+
+import UIKit
+import SnapKit
+
+class DetailViewController: UIViewController {
+
+    var setting: Setting?
```

Comment:
make private let setting: Setting. So you will encapsulate it and prevent from changing outside